packages/client/modules/teamDashboard/components/TeamDashHeader/TeamDashHeader.tsx

```diff
@@ -132,42 +136,55 @@ const TeamDashHeader = (props: Props) => {
     ...(canViewInsights ? [{label: 'Insights', path: 'insights'}] : [])
   ]
 
+  const insightsSeenKey = `insightsSeen_${teamId}`
+  const isLocalStorageAvailable = typeof window !== 'undefined' && window.localStorage
```

Comment:
+1 Any reason we explicitly check this here? I don't see this checked in other places

Reply:
Local storage might not be available, like in Safari's incognito mode. It could throw an error without the check